components/data-services/org.wso2.carbon.dataservices.ui/src/main/java/org/wso2/carbon/dataservices/ui/beans/Result.java

```diff
@@ -221,6 +221,12 @@ public Element removeElement(String elementName) {
 		return element;
 	}
 
+	public void removeXMLOutMappings() {
+		elements.clear();
```

Comment:
Don't use tabs for indentation, use spaces for this, configure the IDE to properly do this.